test/parallel_api/ranges/copy_ranges_factory_sycl.pass.cpp

```diff
@@ -16,13 +16,14 @@
 #include <iostream>
```

Comment:
oneDPL headers should be included before <iostream> as well here and for other ranges tests